StreamChatUITestsAppUITests/MockServer/MessageList.swift

```diff
@@ -28,7 +28,7 @@ extension StreamMockServer {
         try? XCTUnwrap(waitForMessageWithId(id))
     }
     
-    func findMessageByUserId(_ userId: String) -> [String: Any]? {
+    func findMessageByUserId(_ userId: String?) -> [String: Any]? {
```

Comment:
Hmm, but the message has to be authored by someone, right? Allowing to pass `nil` here is a bit confusing